logicrunner/builtin/contract/deposit/deposit.go

```diff
@@ -50,13 +50,13 @@ const (
 type Deposit struct {
 	foundation.BaseContract
 	Balance                 string              `json:"balance"`
-	PulseDepositCreate      insolar.PulseNumber `json:"timestamp"`
-	PulseDepositHold        insolar.PulseNumber `json:"holdStartDate"`
-	PulseDepositUnHold      insolar.PulseNumber `json:"holdReleaseDate"`
-	MigrationDaemonConfirms [3]string           `json:"confirmerReferences"`
-	Amount                  string              `json:"amount"`
-	Bonus                   string              `json:"bonus"`
-	TxHash                  string              `json:"ethTxHash"`
+	PulseDepositCreate      insolar.PulseNumber  `json:"timestamp"`
+	PulseDepositHold        insolar.PulseNumber  `json:"holdStartDate"`
+	PulseDepositUnHold      insolar.PulseNumber  `json:"holdReleaseDate"`
```

Comment:
We can have only PulseDepositUnHold to support our logic according to @papa-vova scheme in confluence. Look at MigrationDeposit.dateMature

Reply:
Delete fields

Reply:
@LinnikD Let's bring the name in line in the next revision - transition to scheme 3?

Reply:
Left one field